webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/ncf/TechniqueWriter.scala

```diff
@@ -193,15 +193,19 @@ class TechniqueWriter (
   }
 
   // Write and commit all techniques files
-  def writeAll(technique : Technique, methods: Map[BundleName, GenericMethod], modId : ModificationId, committer : EventActor) : Result[Seq[String]] = {
+  def writeAll(technique : Technique, methods: Map[BundleName, GenericMethod], modId : ModificationId, committer : EventActor, launchLibUpdate : Boolean) : Result[Seq[String]] = {
```

Comment:
boolean option are hard to understand (we should never have one I believe). I think we should have two methods here: `writeAll` and `writeAllAndUpdateTechLib` and just call the correct one in either place.